### PR TITLE
Pass new tag to version.yaml in terra-helmfile repo

### PR DESCRIPTION
**Pull request by smark88**

### CODE REVIEW

**Comment by smark88** [comment]
Just needs `git push` added

File: .github/workflows/alpha-promotion.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      - name: "Find and replace Datarepo version in versions.yaml"
+        uses: docker://mikefarah/yq:latest
+        with:
+          args: yq w -i terra-helmfile/versions.yaml releases.datarepo.appVersion ${{ steps.apiprevioustag.outputs.tag }}
+      - name: "Find and replace Datarepo chartVersion version in versions.yaml"
+        uses: docker://mikefarah/yq:latest
+        with:
+          args: yq w -i terra-helmfile/versions.yaml releases.datarepo.chartVersion ${{ env.chartVersion }}
+      - name: "Read terra-helmfile daterepo fields in versions.yaml"
+        uses: docker://mikefarah/yq:latest
```
Comment (snf2ye) [inline .github/workflows/alpha-promotion.yaml:40]: Sorry to be lame, but is there a more descriptive docker path we could use?

Comment (smark88) [reply .github/workflows/alpha-promotion.yaml:40]: i don't follow? mikefarah is the official releaser of yq
https://github.com/mikefarah/yq
https://hub.docker.com/r/mikefarah/yq

**Review by fboulnois — APPROVED** [review]
lgtm

File: .github/workflows/alpha-promotion.yaml (hunk 1 above)
Comment (snf2ye) [reply .github/workflows/alpha-promotion.yaml:40]: Gotcha! I was wrong.

**Review by snf2ye — APPROVED** [review]

**Comment by smark88** [comment]
This now creates automerge pr for tracking purposes vs pushing to master

https://github.com/broadinstitute/terra-helmfile/pull/92

@fboulnois @snf2ye please review

**Review by snf2ye — APPROVED** [review]

**Review by fboulnois — APPROVED** [review]

**Pull request merged**